app/controllers/favorites_controller.rb

```diff
@@ -0,0 +1,21 @@
+class FavoritesController < ApplicationController
+
+  def index
+    @favorites = Favorite.all
```

Comment:
shouldn't this be current_user.favorites? Since we only want to display that user's favorites